### DISCUSSION TITLE
Extend duration of selection quickmarks

### DISCUSSION THREAD

**Discussion by jgarte** [meow-edit/meow#455] [Q&A] [answered]
Hi,

How can I increase the duration that the quickmarks stay present in the buffer?

**Comment by DogLooksGood** [comment, marked as answer]
Check this
https://github.com/meow-edit/meow/blob/master/CUSTOMIZATIONS.org#meow-expand-hint-remove-delay

**Reply by jgarte** [reply]
yayyy, `meow-expand-hint-remove-delay` TIL...

I [set](https://git.sr.ht/~whereiseveryone/confetti/commit/8d92bdbd147dfab3dacde1b3b56828a9abb57c3d) it to two seconds, `2.0`:

![output-2023-05-07-23:09:59](https://user-images.githubusercontent.com/47760695/236731979-cb1be411-72c3-47af-b0d3-aec919893499.gif)
